cmd/eksctl-anywhere/cmd/common.go

```diff
@@ -20,7 +20,7 @@ func getImages(clusterSpecPath, bundlesOverride string) ([]v1alpha1.Image, error
 	if err != nil {
 		return nil, err
 	}
-	bundle := clusterSpec.VersionsBundle
```

Comment:
eksctl-anywhere isn't production code as far as I'm aware, but have we validated changing these defaults has no side effects?